docs/plugins/repository-s3.asciidoc

```diff
@@ -184,6 +184,22 @@ pattern then you should set this setting to `true` when upgrading.
     https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Builder.html#disableChunkedEncoding--[AWS
     Java SDK documentation] for details. Defaults to `false`.
 
+`region`::
```

Comment:
I'm wondering if it should be named `signing_region` to make it more explicit?

Reply:
I prefer to use the same naming convention as AWS does (i.e. just plain "region")

Reply:
Ok